publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/helper/RMQHelper.java

```diff
@@ -53,7 +53,7 @@ public void setRabbitMqPropertiesMap(Map<String, RabbitMqProperties> rabbitMqPro
     }
 
     @PostConstruct
-    public void init() {
+    public void init(){
```

Comment:
why is there a change here, we need space after the method declaration, please keep it.